sdk/keyvault/keyvault-common/review/keyvault-common.api.md

```diff
@@ -0,0 +1,41 @@
+## API Report File for "@azure/keyvault-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ChallengeCallbacks } from '@azure/core-rest-pipeline';
+
+// @public
+export function createChallengeCallbacks(options?: CreateChallengeCallbacksOptions): ChallengeCallbacks;
```

Comment:
I was a little concerned about naming being generic. However, this is from a package that for internal consumption so should be fine.

Reply:
still might be worth something like `createKeyVaultAuthChallengeCallbacks`?

Reply:
Yeah, I'm for renaming it. Will need to change it across all the packages in this PR though (may this be the last time that has to be done!)